client/src/Interfaces/post/IAddPost.tsx

```diff
@@ -1,4 +1,5 @@
 interface IAddPost {
+  UserId:number,
```

Comment:
userId camelCase

Reply:
this naming style is related to the interfaces that are used in sequalize. which we had to name that way because of a persisting error that kept popping up if we used camel case